cmd/simpleargs/go.mod

```diff
@@ -0,0 +1,11 @@
+module /home/niki/github.com/prombench/cmd/simpleargs
```

Comment:
is there any reason why you haven't updated the folder structure?

Reply:
I will update it.